src/topology/algebra/module/basic.lean

```diff
@@ -167,10 +168,13 @@ S.to_add_subgroup.topological_add_group
 end submodule
 
 section closure
-variables {R : Type u} {M : Type v}
+variables {R R' : Type u} {M M' : Type v}
```

Comment:
Is it necessary to introduce `M'`? It seems like you could instead add `[module R' M] [has_continuous_smul R' M]`.

Reply:
The point is that `M'` is an `add_comm_group` whereas `M` is an `add_comm_monoid`

Reply:
Oh sorry, I missed that.